README.md

```diff
@@ -57,7 +57,31 @@ provider "instaclustr" {
     api_key = "<Your provisioning API key here>"
 }
 ```
-or just input them when Terraform indicates you.
+
+If you wish to keep secrets in the ENV instead of stored in your main.tf file use the following method:
```

Comment:
maybe say "in your terraform configuration files" because it's not always named `main.tf`

Reply:
agree, fixed